### PR TITLE
Hackerjsimon flower

### PR DESCRIPTION
**Pull request by hackerjsimon**
## About The Pull Request

Adds Bamboo stuff like Spears BLowgun and punji sticks
as well has Tweaking Moonflowers added morphine as well as increasing its rarity upon mutation

## Why It's Good For The Game

Add DLC to botany

Also thank thatlagothere for the amazing sprites
## Changelog
👍 MoonFLower Tweaking
👍 Bamboo tools(I may have to make a second merge soon to fix sprite bugs that may occur hopefully they shouldn't)
/:cl:

### CODE REVIEW

File: code/modules/hydroponics/grown/flowers.dm [hunk 1]
```diff
@@ -131,7 +131,7 @@
 // Moonflower
 /obj/item/seeds/sunflower/moonflower
 	name = "pack of moonflower seeds"
-	desc = "These seeds grow into moonflowers."
+	desc = "its petals are known for helping insomiacs around the world."
```
Comment (qwertyquerty) [inline code/modules/hydroponics/grown/flowers.dm]: Capitalize

**Comment by qwertyquerty** [comment]
Is this ported from somewhere?

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 2]
```diff
@@ -321,6 +322,33 @@ GLOBAL_LIST_INIT(cloth_recipes, list ( \
...
+
+/obj/item/stack/sheet/mineral/bamboo
+	name = "bamboo cuttings"
+	desc = "Finely cut bamboo sticks."
+	singular_name = "cut bamboo"
+	icon_state = "sheet-bamboo"
+	item_state = "sheet-bamboo"
+	icon = 'icons/obj/stack_objects.dmi'
+	sheettype = "bamboo"
+	throwforce = 15
```
Comment (qwertyquerty) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: This does 3 more than the force of a toolbox on impact. Seems kind of off

**Comment by hackerjsimon** [comment]
Ay school its ported from tg

File: code/game/objects/items/stacks/sheets/sheet_types.dm (hunk 2 above)
Comment (miniusAreas) [reply code/game/objects/items/stacks/sheets/sheet_types.dm]: IRL, bamboo hurts. Think about it, it's a long wooden pole that splinters easly. Why dafuq do you thing punji sticks exsist?

Comment (qwertyquerty) [reply code/game/objects/items/stacks/sheets/sheet_types.dm]: More than getting hit over the head with a metal toolbox?

**Comment by hackerjsimon** [comment]
sorry i was at school

File: code/game/objects/items/stacks/sheets/sheet_types.dm (hunk 2 above)
Comment (hackerjsimon) [reply code/game/objects/items/stacks/sheets/sheet_types.dm]: Fixed

**Comment by qwertyquerty** [comment]
not what I meant for you to do, I'll fix it

**Pull request merged**